traffic_portal/app/src/modules/public/login/login.tpl.html

```diff
@@ -47,6 +48,13 @@ <h2>Login</h2>
                         </div>
                     </div>
                 </form>
+                <form name="loginForm" role="form" class="form-horizontal form-label-left" novalidate ng-if="oAuthEnabled">
```

Comment:
can you rename this to name="ssoLoginForm" or something so there's not 2 forms of the same name?

Reply:
done